### ISSUE TITLE
ApiUseTags decorator don't work on Method

### ISSUE THREAD

**Issue by miramo** [nestjs/swagger#35]
## I'm submitting a...
<pre><code>
[ ] Regression 
[X] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Minimal reproduction of the problem with instructions
I'm trying to tag just on one method of my controller but it seems not working.

(On the doc: ApiUseTags() | Method / Controller)

```typescript
import { Get, Controller } from '@nestjs/common';
import { ApiUseTags } from '@nestjs/swagger';

@Controller()
export class AppController {
  @Get()
  @ApiUseTags('admin') // there is no effect in swagger
  ping(): string {
    return 'pong';
  }
}
```

But on controller this works perfect.

## Environment

<pre><code>
@nestjs/common@^4.5.9
@nestjs/core@^4.5.10
@nestjs/swagger@^1.1.4
reflect-metadata@^0.1.10
 
For Tooling issues:
- Node version: v8.9.4
- Platform:  Mac

Others:
- yarn
</code></pre>

**Comment by ericzon** [comment]
I've just tested this and experienced the same.

**Comment by kamilmysliwiec** [comment]
Fixed due to #79 and published as a `v1.3.0` already.
